drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2027,7 +2027,7 @@ void RasterizerSceneGLES3::render_scene(RID p_render_buffers, const CameraData *
 	bool keep_color = false;
 
 	if (get_debug_draw_mode() == RS::VIEWPORT_DEBUG_DRAW_OVERDRAW) {
-		clear_color = Color(0, 0, 0, 1); //in overdraw mode, BG should always be black
+		clear_color = Color::BLACK; //in overdraw mode, BG should always be black
```

Comment:
```suggestion
		clear_color = Color::BLACK; // In overdraw mode, BG should always be black.
```
You could fix the comment if you are editing this.